src/components/UI/Display/Apps/AppList/Icon.tsx

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ImgWithFallback from 'shared/ImgWithFallback';
+import styled from 'styled-components';
+import theme from 'styles/theme';
+import CachingColorHash from 'utils/cachingColorHash';
+
+const colorHash = new CachingColorHash();
+
+function acronymize(text: string) {
```

Comment:
Would be awesome to memoize this. Maybe in a future PR

Reply:
I did something wrong in this PR :P these files were renamed, but when I did a rebase I committed both versions of them. 😅 (Also applied to:  src/components/UI/Display/Apps/AppDetailsNew/utils.js)

but yeah, that can totally be memoized :+1: will note it down.

will delete these two files though since they should not be getting referenced anywhere. thx for noticing!!